### PR TITLE
Add option to list local playlists

### PR DESCRIPTION
**Pull request by academo**
Simple initial functionality to list playlists currently stored in your disk.

The way gpymusic stores playlists are files inside the playlists folder, currently there is no way to know what playlists are available other than your own brain.

This simple functionality will list the files that exist on the playlists folder and nothing else.

### CODE REVIEW

**Comment by christopher-dG** [comment]
Hey there, thanks for these PRs! I'm quite busy this week so I'll get around to reviewing them on the weekend 😊.

**Comment by christopher-dG** [comment]
I think this command might be better named `playlists`, since `list` isn't very specific (search technically lists things too...). Also the updated readme has `l/list` but there's no `l` alias. I would probably only include the long-form command.

**Comment by academo** [comment]
@christopher-dG pushed some changes regarding your comments

File: gpymusic/client.py [hunk 1]
```diff
@@ -139,6 +142,25 @@ def restore(self, fn=None):
...
+            msg  = "\n==Local playlists available==\n\n"
+            for item in playlists:
+                msg = msg + " - " + item + "\n"
+            msg = msg + "\n\nUse read [playlist-name] to show playlist"
+
+            common.w.main.addstr(msg)
+            common.w.main.refresh()
+
+        else:
+            common.w.outbar_msg('No local playlist found')
```
Comment (christopher-dG) [inline gpymusic/client.py:162]: playlist -> playlists

File: gpymusic/client.py [hunk 2]
```diff
@@ -139,6 +142,25 @@ def restore(self, fn=None):
...
 
+    def playlists(self, arg=None):
+        path = join(common.DATA_DIR, 'playlists')
+        playlists = listdir(path)
+        if len(playlists):
+            common.v.clear()
+            common.w.main.erase()
+            common.w.outbar_msg('Listing local playlists')
+
+            msg  = "\n==Local playlists available==\n\n"
```
Comment (christopher-dG) [inline gpymusic/client.py:153]: There are two spaces between `msg` and `=`. Also please make these strings single-quoted.

File: gpymusic/client.py [hunk 3]
```diff
@@ -139,6 +142,25 @@ def restore(self, fn=None):
             else:
                 common.q.restore(json_songs)
 
+    def playlists(self, arg=None):
+        path = join(common.DATA_DIR, 'playlists')
+        playlists = listdir(path)
+        if len(playlists):
```
Comment (christopher-dG) [inline gpymusic/client.py:148]: You can just use `if playlists` here

File: gpymusic/client.py [hunk 4]
```diff
@@ -139,6 +142,25 @@ def restore(self, fn=None):
             else:
                 common.q.restore(json_songs)
 
+    def playlists(self, arg=None):
```
Comment (christopher-dG) [inline gpymusic/client.py:145]: Add a docstring here, similar to the other methods

File: gpymusic/client.py [hunk 5]
```diff
@@ -139,6 +142,25 @@ def restore(self, fn=None):
...
+        path = join(common.DATA_DIR, 'playlists')
+        playlists = listdir(path)
+        if len(playlists):
+            common.v.clear()
+            common.w.main.erase()
+            common.w.outbar_msg('Listing local playlists')
+
+            msg  = "\n==Local playlists available==\n\n"
+            for item in playlists:
```
Comment (christopher-dG) [inline gpymusic/client.py:154]: This could be simplified as: `msg += '\n'.join('- ' + p for p in playlists)`

File: gpymusic/client.py [hunk 6]
```diff
@@ -139,6 +142,25 @@ def restore(self, fn=None):
...
+                msg = msg + " - " + item + "\n"
+            msg = msg + "\n\nUse read [playlist-name] to show playlist"
```
Comment (christopher-dG) [inline gpymusic/client.py:156]: `msg +=` here as well

**Pull request closed without merging**